debezium-connector-mongodb/pom.xml

```diff
@@ -90,25 +105,17 @@
         </dependency>
     </dependencies>
     <properties>
-        <!-- By default we need to run these 5 containers, although the two init containers exit quickly -->
-        <docker.image>mongo1,mongo-init</docker.image>
-
-        <!-- The command line args passed to each mongod service -->
-        <mongo.data.replicaset.name>rs0</mongo.data.replicaset.name>
-        <mongo.cfg.replicaset.name>cfg</mongo.cfg.replicaset.name>
-        <mongo.data.runOptions>--replSet ${mongo.data.replicaset.name} --oplogSize=2 --enableMajorityReadConcern</mongo.data.runOptions>
-        <mongo.cfg.runOptions>--configsvr --replSet ${mongo.cfg.replicaset.name} --oplogSize=2 --enableMajorityReadConcern</mongo.cfg.runOptions>
-        <mongo.router.runOptions>mongos --configdb ${mongo.cfg.replicaset.name}/config:27019</mongo.router.runOptions>
         <!-- 
         Set this property to 'true' in a profile to skip the integration tests and not even run the Docker containers.
         -->
         <docker.skip>false</docker.skip>
         <docker.exposeContainerInfo>docker.container</docker.exposeContainerInfo>
-        <docker.dbs>mongo:${version.mongo.server},debezium/mongo-initiator:${version.mongo.server}</docker.dbs>
-        <docker.filter>${docker.dbs}</docker.filter>
         <!-- Apicurion container properties -->
         <apicurio.port>8080</apicurio.port>
         <apicurio.init.timeout>60000</apicurio.init.timeout> <!-- 60 seconds -->
+        <!-- Mongo test containers -->
+        <mongodb.docker.desktop.ports>27017,27018,27019</mongodb.docker.desktop.ports>
+        <mongodb.replica.size>1</mongodb.replica.size>
```

Comment:
@jpechane  what should be the dealt here? We can either have `1` which would mean that locally one will have to think about it or we can keep the `3` and set the property in GH action definition.